src/features/bookings/components/TicketBody/ticketBodyMessages.test.ts

```diff
@@ -0,0 +1,64 @@
+import mockdate from 'mockdate'
+
+import { WithdrawalTypeEnum } from 'api/gen'
+import {
+  getDelayMessage,
+  getEmailMessage,
+  getStartMessage,
+} from 'features/bookings/components/TicketBody/ticketBodyMessages'
+import { formatSecondsToString } from 'features/bookings/helpers'
+
+const whiteSpace = ' '
+const lineBreak = '\n'
+const today = new Date(2020, 11, 17)
+const tomorrow = new Date(2020, 11, 18)
+mockdate.set(today)
+
+describe('<ticketBodyMessages/>', () => {
+  describe('getStartMessage', () => {
+    it('should return the correct message if withdrawal type is on site', () => {
```

Comment:
je ne sais pas s'il est vraiment nécessaire de tester toutes ces fonctions unitairement alors que tu as déjà testé ailleurs

`return` c'est technique comme terme

`the correct message` je trouve peu utile comme information, ça ne ressemble pas trop à une règle

Reply:
Je trouve que ces tests sont sécurisants dans le sens où il y avait pas mal de conditions d'affichage possibles

Reply:
si quelqu'un modifie les tests, iel doit vérifier que rien n'est cassé et que les règles d'avant sont toujours testées